titanic.py

```diff
@@ -93,12 +93,46 @@ def maximum_energy_delta(bqm):
                      for j in bqm.iter_neighbors(i))
                for i in bqm.iter_variables())
 
-def run_demo():
+def mutual_information_bqm(dataset, features):
+    """Build a QUBO that maximizes MI between survival and a subset of features"""
```

Comment:
It's not a big deal, but it might be more accurate (and more consistent with the function name) to use BQM here instead of QUBO.  Thinking being that the BQM is an encoding of the QUBO for use on the D-Wave system: https://docs.ocean.dwavesys.com/en/stable/concepts/bqm.html.  @JoelPasvolsky might have an opinion on this.
```suggestion
    """Build a BQM that maximizes MI between survival and a subset of features"""
```

Reply:
The most important thing is consistency within a project. We tend to use QUBO to describe a BQM that has `{0,1}` variables and `Ising` to describe a BQM that has `{-1,+1}` variables. Adding confusion is that we also use them to describe a data representation. I.e. BQM is the object, QUBO is represented by a dict `{(u, u): b0, (u, v): b1,...}` and Ising by two dicts `{u: bu,...}`, `{(u, v): buv,...}`. In this case because the function is returning a BQM object I would agree we should say BQM.

Reply:
If we want to be more specific, we can say "binary BQM" or something like that.

Reply:
done